EIPS/eip-5005.md

```diff
@@ -12,12 +12,12 @@ requires: 165
 ---
 
 ## Abstract
-ERC-5005 (Zodiac Avatar Accounts) is a philosophy and open standard for composable and interoperable tooling for programmable Ethereum accounts. Zodiac-compatible tooling separates the account taking actions/holding tokens (known as the "avatar") and the authorization logic into two (or more) separate contracts. This standard defines the `IAvatar` interface, to be implemented by avatar contracts, while the authorization logic can be implemented with any combination of other tools (for example, DAO tools and frameworks).
+ERC-5005 (Zodiac Modular Accounts) is an open standard for composable and interoperable tooling for programmable Ethereum accounts. Compatible tooling separates the account taking actions/holding tokens (known as the "avatar") and the authorization logic into two (or more) separate contracts. This standard defines the `IAvatar` interface, to be implemented by avatar contracts, while the authorization logic can be implemented with any combination of other tools (for example, DAO tools and frameworks).
```

Comment:
```suggestion
This EIP (Zodiac Modular Accounts) is an open standard for composable and interoperable tooling for programmable Ethereum accounts. Compatible tooling separates the account taking actions/holding tokens (known as the "avatar") and the authorization logic into two (or more) separate contracts. This standard defines the `IAvatar` interface, to be implemented by avatar contracts, while the authorization logic can be implemented with any combination of other tools (for example, DAO tools and frameworks).
```

Reply:
The "commit suggestion" button on GitHub didn't work, so I just pushed the changes.